web/skins/classic/includes/export_functions.php

```diff
@@ -263,12 +263,12 @@ function exportEventImages( $event, $exportDetail, $exportFrames, $myfilelist )
 <div id="imagevideo" align="center"></div>
 <br>
 <div align="center">
-<button onclick="stepbackward()">< Step</button><button 
-	id="btnrwd" onclick="rewind()" >Rwd</button><button
-	id="btnplay" onclick="playstop()">Stop</button><button
-	onclick="stepforward()">Step ></button><button
-	id="btnspeedup" onclick="speedup()">speedup</button><button
-	id="btnspeeddown" onclick="speeddown()">slowdown</button>
+<button data-on-click="stepbackward">< Step</button><button 
```

Comment:
While we are in here, should that < Step be &lt; Step?

Reply:
ahem... I mean & lt ;

Reply:
Yes, definitely. I'll make a PR